Cargo.toml

```diff
@@ -66,7 +66,7 @@ failure_derive = { version = "0.1.5", optional = true}
 [dev-dependencies]
 pretty_assertions = "0.6.1"
 tempfile = "^3.1.0"
-criterion = "0.2"
+criterion = "0.3.0"
```

Comment:
> Ah, nice -- what did you make bump criterion? (just curious!).

criterion 0.3.0 has the grouping functionality, so that we can iterate through several variables and combine them in one plot. Unfortunately this version also sets the minimum of the benchmark sample size to 10, before it was possible to go lower.

> ps. It looks like you committed .DS_Store inside this PR -- I suggest we extend the .gitignore to ignore these kind of files 😉

I accidentally committed several files that were supposed to stay local. I will remove those (eg `playground.rs`) and also add `.DS_Store` to `.gitignore`